pkg/nfd-topology-updater/nfd-topology-updater.go

```diff
@@ -294,6 +294,7 @@ func (w *nfdTopologyUpdater) updateNodeResourceTopology(zoneInfo v1alpha2.ZoneLi
 		nrtNew := v1alpha2.NodeResourceTopology{
 			ObjectMeta: metav1.ObjectMeta{
 				Name:            w.nodeName,
+				Labels:          map[string]string{"owner-pod": os.Getenv("POD_NAME")},
```

Comment:
why `owner-pod` ? use const? Pod names are only unique in the namespace not across whole K8s cluster.